policy/modules/admin/usermanage.te

```diff
@@ -277,6 +277,12 @@ optional_policy(`
 	rpm_rw_pipes(groupadd_t)
 ')
 
+ifdef(`init_systemd',`
```

Comment:
I don't think this needs to be in a init_systemd block; the optional should suffice.

Reply:
Ok. I removed the `ifdef(init_systemd)` block.